client/src/containers/CampaignAdmin/tests/CampaignAdminItem-test.js

```diff
@@ -0,0 +1,149 @@
+/* global jest, describe, beforeEach, it, expect */
+
+jest.unmock('react');
+jest.unmock('react-addons-test-utils');
+jest.unmock('../CampaignAdminItem');
+
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import CampaignAdminItem from '../CampaignAdminItem';
+
+describe('CampaignAdminItem', () => {
+  let props = null;
+
+  beforeEach(() => {
+    props = {
+      campaign: {},
+      item: { ID: 1 },
+    };
+  });
+
+  describe('Links', () => {
+    let cmp = null;
+    let tooltip = null;
+
+    beforeEach(() => {
+      cmp = ReactTestUtils.renderIntoDocument(
+        <CampaignAdminItem {...props} />
+      );
+      tooltip = ReactTestUtils
+        .scryRenderedDOMComponentsWithClass(cmp, 'campaign-admin__item-links');
+    });
+
+    it('should not show link icon by default', () => {
+      expect(tooltip).toHaveLength(0);
+    });
+
+    it('should show link icon when the item is being from the selected item', () => {
+      props.isLinked = true;
+      cmp = ReactTestUtils.renderIntoDocument(
+        <CampaignAdminItem {...props} />
+      );
+      tooltip = ReactTestUtils
+        .scryRenderedDOMComponentsWithClass(cmp, 'campaign-admin__item-links');
+
+      expect(tooltip).toHaveLength(1);
+      expect(tooltip[0]).toBeInstanceOf(HTMLElement);
```

Comment:
That's cool, could be renamed, but I was more looking for a more robust unit test.
So rather than checking for a generic `HTMLElement` which can contain anything, look for a bit more specifically the first item title that's supposed to be rendered.